docs/daily-usage.md

```diff
@@ -93,10 +93,17 @@ Start Odoo with:
 invoke start
 ```
 
+You can also apply some common configurations to help you setup the devel environment
+(like setting the `report.url` parameter)::
+
+```bash
+invoke preparedb
+```
+
 All of the above in one shot:
 
 ```bash
-invoke img-pull img-build --pull git-aggregate resetdb start
+invoke img-pull img-build git-aggregate resetdb start
```

Comment:
Revert this, please, it's not the same. `inv img-build --pull` pulls the dockerfile dependencies.

Reply:
Yes, but `--pull` is true by default: https://github.com/Tecnativa/doodba-copier-template/blob/stable/tasks_downstream.py#L387

Reply:
🤦🏼‍♂️ true